packages/firestore/test/lite/integration.test.ts

```diff
@@ -2038,3 +2045,142 @@
     });
   });
 });
+
+describe('countQuery()', () => {
```

Comment:
Can you write a simpler converter? Or perhaps re-use an existing converter defined somewhere else? Since the converter is never actually used for anything, keeping it as simple as possible improves readability by removing distractions.